spec/features/authentication_pages_spec.rb

```diff
@@ -52,40 +56,50 @@
         it { should have_content('Welcome to Twitter Clone') }
         it { should have_link('Sign in') }
         it { should have_link('Sign up') }
+
+        describe 'screenshot', js: true do
+          it { page.save_screenshot "sign-in.png" }
+        end
       end
 
       describe "when attempting to visit a protected page" do
-        before do
-          visit edit_user_path(user)
-          fill_in "Email",    with: user.email
-          fill_in "Password", with: user.password
-          click_button "Sign in"
+        describe "visiting the edit page" do
+          before { visit edit_user_path(user) }
+          it { should have_content('Sign in') }
         end
 
         describe "after signing in" do
+          before do
+            visit edit_user_path(user)
+            fill_in "Email",    with: user.email
+            fill_in "Password", with: user.password
+            click_button "Sign in"
+          end
+
           it "should render the desired protected page" do
-            expect(page).to have_content('Update your profile')
+            should have_content('Update your profile')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.